lms/envs/common.py

```diff
@@ -3390,3 +3390,6 @@
     'sequential',
     'vertical',
 }
+
+############### Settings for anonymizing users data in the forum ###
+FORUM_MONGO_URL = 'mongodb://localhost:27017/cs_comments_service'
```

Comment:
You are setting this default, but the setting isn't really being loaded from ENV_TOKENS, so it will never get the actual value. Look at aws.py to see how to load settings from the env.* files.